modules/runtime/pages/performance-tuning.adoc

```diff
@@ -399,6 +399,13 @@ Notably, we recommend you to set the initial (`-Xms`) and maximum (`-Xmx`) heap
 This reduces the likelihood of the JVM garbage collector starting. +
 While the garbage collector is running, it prevents creation of new objects, which slows down the application server.
 
+===== Garbage Collector
+We recommend to update the Garbage Collector parameters to use the following ones:  +
```

Comment:
Maybe explain why? What a user is going to gain by doing this?

Reply:
this will be improved in #1855